test/close-pipelining.test.js

```diff
@@ -5,9 +5,7 @@ const test = t.test
 const Fastify = require('..')
 const autocannon = require('autocannon')
 
-// this tests on windows takes an unusually large amount of time.
-// https://github.com/fastify/fastify/issues/2470
-t.setTimeout(45000)
+t.jobs = 2
```

Comment:
This assumes there are at least two CPU cores, or threads, available in the environment. I also think this is just a different variation of hiding the problem.